examples/functions/functions.md

```diff
@@ -38,7 +38,9 @@ fn main() {
 }
 
 fn foo(test ...string) {
-	println(test)
+	for txt in test {
+		println(txt)
+	}
```

Comment:
It's interesting how this spaces are different than the struct indentation below.

Reply:
We're constantly have problems with that in V. How could we solve this? Thoughts?

Reply:
I think 4 spaces is better